_team/_defaults.md

```diff
@@ -1,6 +1,7 @@
 ---
 name:
 position:
+affiliation:
```

Comment:
Is this a required field? As in what happens with folks that don't have one at 

Also I'm not seeing a staff affiliation for myself.

Reply:
Added yours, apologies! 

And not a required field - if it's not present it's just skipped.